AutomationPractice/features/Pages/AuthencationPage.py

```diff
@@ -1,15 +1,19 @@
 import Locator
```

Comment:
The Locator does not belong to any package. Please move it into features package.

Reply:
fixed as above cmt